lib/init-mongo.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
This is the script that gets copied into the mongo container and initializes mongodb with users, database, and collection.